src/main/java/newbank/server/exceptions/AccountBalanceInsufficientException.java

```diff
@@ -0,0 +1,27 @@
+package newbank.server.exceptions;
+
+import org.javamoney.moneta.Money;
+
+public class AccountBalanceInsufficientException extends Exception {
+  private static final long serialVersionUID = 8234423235864929105L;
+
+  private Money requiredBalance;
+  private Money currentBalance;
+
+  public AccountBalanceInsufficientException(Money requiredBalance, Money currentBalance) {
+    this.requiredBalance = requiredBalance;
+    this.currentBalance = currentBalance;
+  }
+
+  public Money getRequiredBalance() {
+    return requiredBalance;
+  }
+
+  public Money getCurrentBalance() {
+    return currentBalance;
+  }
```

Comment:
These are never called. More importantly though, why not reuse the `AccountInvalidException` Jules created for the `PAY` command?

Reply:
Hi Zac. AccountInvalid is used when an account is not found, a very different exception.

Reply:
Indeed, these 2 can be probably removed in the spirit of YAGNI.

Reply:
Nothing to do here...